Tests/SentryTests/Integrations/SentryCrash/SentryCrashIntegrationTests.swift

```diff
@@ -216,21 +216,6 @@ class SentryCrashIntegrationTests: XCTestCase {
         XCTAssertTrue(fixture.sentryCrash.closeCalled)
     }
     
-    func testUninstall_DoesNotUpdateLocale_OnLocaleDidChangeNotification() {
```

Comment:
`m`: Why did you remove these tests?

Reply:
Because I removed the locale property, which needs to be restored. So I'll restore the tests as well.